_topic_maps/_topic_map.yml

```diff
@@ -191,10 +191,8 @@ Topics:
 - Name: Managing Compliance
   Dir: manage-compliance
   Topics:
-  - Name: Managing the compliance 1.0 feature
-    File: managing-compliance-10
-  - Name: Managing the compliance 2.0 feature (Technology preview)
-    File: managing-compliance-20
+  - Name: Managing the compliance feature
```

Comment:
**thought:** 💭 Since we are removing the two items from the list, we can eliminate the category and have a single "Manage Compliance page" instead.

```
---
Name: Operating
Dir: operating
Distros: openshift-acs
Topics:
- Name: Viewing the dashboard
  File: view-dashboard
- Name: Using the Compliance Operator
  Dir: manage-compliance-operator
  Topics:
  - Name: Using Compliance Operator with RHACS
    File: compliance-operator-rhacs
- Name: Managing Compliance
  File: manage-compliance
  ```
Of course, we must delete the Manage Compliance directory, move the `manage-compliance` file into the Operating directory, and create redirects for the changes.